docs/src/token.md

```diff
@@ -381,28 +388,43 @@ RPC response error -32002: Transaction simulation failed: Error processing Instr
 
 But repeating with a second multisig signer, succeeds
 ```
-spl-token mint 4VNVRJetwapjwYU8jf4qPgaCeD76wyz8DuNj8yMCQ62o 1 EX8zyi2ZQUuoYtXd4MKmyHYLTjqFdWeuoTHcsTdJcKHC \
+$ spl-token mint 4VNVRJetwapjwYU8jf4qPgaCeD76wyz8DuNj8yMCQ62o 1 EX8zyi2ZQUuoYtXd4MKmyHYLTjqFdWeuoTHcsTdJcKHC \
 --owner 46ed77fd4WTN144q62BwjU2B3ogX3Xmmc8PT5Z3Xc2re \
 --multisig-signer signer-1.json \
 --multisig-signer signer-2.json
+```
+```console
 Minting 1 tokens
   Token: 4VNVRJetwapjwYU8jf4qPgaCeD76wyz8DuNj8yMCQ62o
   Recipient: EX8zyi2ZQUuoYtXd4MKmyHYLTjqFdWeuoTHcsTdJcKHC
 Signature: 2ubqWqZb3ooDuc8FLaBkqZwzguhtMgQpgMAHhKsWcUzjy61qtJ7cZ1bfmYktKUfnbMYWTC1S8zdKgU6m4THsgspT
 ```
 
-### Example: Mint with multisig authority and offline signers
+### Example: Offline signing with multisig 
 
-This example builds off of the [online mint with multisig](#example-mint-with-multisig-authority)
-example. Be sure to familiarize yourself with that process, [offline signing](https://docs.solana.com/offline-signing),
-and the [durable nonce](https://docs.solana.com/offline-signing/durable-nonce)
-feature before continuing.
+Sometimes online signing is not possible or desireable. Such is the case for example when signers are not in the same geographic location  
+or when they use air-gapped devices not connected to the network.  In this case, we use offline signing which combines the   
+previous examples of [multisig](#example-mint-with-multisig-authority) with [offline signing](https://docs.solana.com/offline-signing)
+and a [nonce account](https://docs.solana.com/offline-signing/durable-nonce).
 
 This example will use the same mint account, token account, multisig account,
-and multisig signer-set keypair filenames as the online example.
+and multisig signer-set keypair filenames as the online example, as well as a nonce
```

Comment:
This was actually omitted intentionally, to ensure that the reader goes and reads about the durable nonce feature before continuing.  It is more important that they understand how that feature works than have a nice copypasta script for using multisig

Reply:
you are right that the changes make it too easy for the newcomer to skip over understanding nonce feature and thus fail to grasp a full understanding of multisig in Solana.   Still, i want to make this as followable as possible for everyone while keeping with the how-to nature of the page.    Let me think on this.   Perhaps an additional paragraph imploring the reader to read up before continuing will work.